groovy/inductor_images.groovy

```diff
@@ -0,0 +1,207 @@
+        TORCH_VISION_BRANCH = params.TORCH_VISION_BRANCH
+    }
+}
+echo "TORCH_VISION_BRANCH: $TORCH_VISION_BRANCH"
+
+TORCH_VISION_COMMIT = ''
+if ('TORCH_VISION_COMMIT' in params) {
+    echo "TORCH_VISION_COMMIT in params"
+    if (params.TORCH_VISION_COMMIT != '') {
+        TORCH_VISION_COMMIT = params.TORCH_VISION_COMMIT
+    }
+}
+echo "TORCH_VISION_COMMIT: $TORCH_VISION_COMMIT"
+
+TORCH_TEXT_BRANCH = ''
+if ('TORCH_TEXT_BRANCH' in params) {
+    echo "TORCH_TEXT_BRANCH in params"
+    if (params.TORCH_TEXT_BRANCH != '') {
+        TORCH_TEXT_BRANCH = params.TORCH_TEXT_BRANCH
+    }
+}
+echo "TORCH_TEXT_BRANCH: $TORCH_TEXT_BRANCH"
+
+TORCH_TEXT_COMMIT = ''
+if ('TORCH_TEXT_COMMIT' in params) {
+    echo "TORCH_TEXT_COMMIT in params"
+    if (params.TORCH_TEXT_COMMIT != '') {
+        TORCH_TEXT_COMMIT = params.TORCH_TEXT_COMMIT
+    }
+}
+echo "TORCH_TEXT_COMMIT: $TORCH_TEXT_COMMIT"
+
+TORCH_AUDIO_BRANCH = ''
+if ('TORCH_AUDIO_BRANCH' in params) {
+    echo "TORCH_AUDIO_BRANCH in params"
+    if (params.TORCH_AUDIO_BRANCH != '') {
+        TORCH_AUDIO_BRANCH = params.TORCH_AUDIO_BRANCH
+    }
+}
+echo "TORCH_AUDIO_BRANCH: $TORCH_AUDIO_BRANCH"
+
+TORCH_AUDIO_COMMIT = ''
+if ('TORCH_AUDIO_COMMIT' in params) {
+    echo "TORCH_AUDIO_COMMIT in params"
+    if (params.TORCH_AUDIO_COMMIT != '') {
+        TORCH_AUDIO_COMMIT = params.TORCH_AUDIO_COMMIT
+    }
+}
+echo "TORCH_AUDIO_COMMIT: $TORCH_AUDIO_COMMIT"
+
+TORCH_BENCH_BRANCH = ''
+if ('TORCH_BENCH_BRANCH' in params) {
+    echo "TORCH_BENCH_BRANCH in params"
+    if (params.TORCH_BENCH_BRANCH != '') {
+        TORCH_BENCH_BRANCH = params.TORCH_BENCH_BRANCH
+    }
+}
+echo "TORCH_BENCH_BRANCH: $TORCH_BENCH_BRANCH"
+
+TORCH_BENCH_COMMIT = ''
+if ('TORCH_BENCH_COMMIT' in params) {
+    echo "TORCH_BENCH_COMMIT in params"
+    if (params.TORCH_BENCH_COMMIT != '') {
+        TORCH_BENCH_COMMIT = params.TORCH_BENCH_COMMIT
+    }
+}
+echo "TORCH_BENCH_COMMIT: $TORCH_BENCH_COMMIT"
+
+TORCH_DATA_BRANCH = ''
+if ('TORCH_DATA_BRANCH' in params) {
+    echo "TORCH_DATA_BRANCH in params"
+    if (params.TORCH_DATA_BRANCH != '') {
+        TORCH_DATA_BRANCH = params.TORCH_DATA_BRANCH
+    }
+}
+echo "TORCH_DATA_BRANCH: $TORCH_DATA_BRANCH"
+
+TORCH_DATA_COMMIT = ''
+if ('TORCH_DATA_COMMIT' in params) {
+    echo "TORCH_DATA_COMMIT in params"
+    if (params.TORCH_DATA_COMMIT != '') {
+        TORCH_DATA_COMMIT = params.TORCH_DATA_COMMIT
+    }
+}
+echo "TORCH_DATA_COMMIT: $TORCH_DATA_COMMIT"
+
+BENCH_COMMIT = ''
+if ('BENCH_COMMIT' in params) {
+    echo "BENCH_COMMIT in params"
+    if (params.BENCH_COMMIT != '') {
+        BENCH_COMMIT = params.BENCH_COMMIT
+    }
+}
+echo "BENCH_COMMIT: $BENCH_COMMIT"
+
+inductor_tools_branch = ''
+if ('inductor_tools_branch' in params) {
+    echo "inductor_tools_branch in params"
+    if (params.inductor_tools_branch != '') {
+        inductor_tools_branch = params.inductor_tools_branch
+    }
+}
+echo "inductor_tools_branch: $inductor_tools_branch"
+
+tag = ''

```

Comment:
We can give a default value, e.g. `test`, in case someone give space value for it

Reply:
OK